models/netsuite2.yml

```diff
@@ -36,9 +37,11 @@ models:
   - name: int_netsuite2__tran_with_converted_amounts
     description: >
       (Step 4/4) Now that we have the exchange rates and the unconverted amounts, the next step is 
-      to calculate the converted total.
-      Additionally, we add in a couple of extra fields that will help us in our final balance sheet 
-      and income statement queries.
+      to calculate the converted total. Additionally, we add in a couple of extra fields that will 
+      help us in our final balance sheet and income statement queries.
+            
+      Note: This is only applicable if you're using exchange rates. See the README for instructions
```

Comment:
could be good to mention the specific `netsuite2__using_exchange_rate` variable so they know where to look in the README